android/assets/jsons/Civ V - Gods & Kings/Units.json

```diff
@@ -1658,6 +1658,17 @@
             "Great Person - [War]", "Unbuildable", "Uncapturable"],
 		"movement": 5
 	},
+	{
+		"name": "Great Admiral",
+		"unitType": "Civilian Water",
+		"uniques": [
+            // TODO: "[Every adjacent [{Friendly} {Water}] unit] heals [100] HP <by consuming this unit>",
+            "[+15]% Strength bonus for [{Military} {Water}] units within [2] tiles",
+			"Can be earned through combat",
+            "Is part of Great Person group [Admiral]",
```

Comment:
One concern I have is the great person group. Is this correct?

Reply:
Should be